app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -112,7 +112,6 @@ private void listenToModelChanges() {
 
         attendeeListener.getNotifier()
             .filter(attendeeModelChange -> attendeeModelChange.getAction().equals(BaseModel.Action.UPDATE))
-            .distinctUntilChanged()
```

Comment:
Yes, the current code is faulty, it'll filter the updates, that's why I asked to add distinctUntilChanged after the map operation. Then it should work fine.

The problem is not of sending the same model again and again, but about the granular changes which will not affect the UI causing it to trigger update again and again.